movie_REST_API/movie_REST_API/ViewModel/MovieViewModel.swift

```diff
@@ -0,0 +1,46 @@
+// MovieViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Получение данных о фильмах
+final class MovieViewModel {
```

Comment:
```suggestion
final class MovieView {
```
Model - вот так вот не надо)